lib/rules/require-padding-newlines-before-export.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Requires newline before module.exports
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requirePaddingNewLinesBeforeExport": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * var a = 2;
+ *
+ * module.exports = a;
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * var a = 2;
+ * module.exports = a;
+ * ```
+ */
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(value) {
+        assert(
+            value === true,
+            this.getOptionName() + ' option requires a true value or should be removed'
+        );
+    },
+
+    getOptionName: function() {
+        return 'requirePaddingNewLinesBeforeExport';
```

Comment:
Fixing these rules should be done in a separate PR then and will likely need to deprecate these ones so we don't break backwards compatibility, right?